tests/scenarios/personas.spec.js

```diff
@@ -0,0 +1,112 @@
+const server = require('apostrophe-nightwatch-tools/server');
+const steps = require('apostrophe-nightwatch-tools/steps');
+
+module.exports = Object.assign(
+  {
+    before: (client, done) => {
+      client.resizeWindow(1200, 800);
+      if (!this._server) {
+        this._server = server.create('localhost', 3111);
+        this._server.start(done);
+      }
+    },
+    after: (client, done) => {
+      client.end(() => {
+        this._server.stop(done);
+      });
+    },
+  },
+  steps.main(),
+  steps.login(),
+  steps.switchLocale('en'),
+  steps.switchToDraftMode(),
+  steps.makeSubPage('Personas Test'),
+  steps.addTextWidgetTo({selector: '.demo-main', text: 'Rich Text Widget Robot Line'}),
+  steps.submitChanges(),
+  {
+    'Switch to robot persona': function(client) {
+      const personaSelect = '.demo-main .apos-area-widget-controls .apos-button[name=personas]';
+      const testPersonaValue = `${personaSelect} option[value="robot"]`;
+      client.moveToElement(personaSelect, 0, 0);
+      client.clickWhenReady(personaSelect);
+      client.clickWhenReady(testPersonaValue);
+      client.pause(500);
+    }
+  },
+  steps.commitAndExport(),
+  {
+    'add video widget': function(client) {
+      const selector = '.demo-main .apos-area-widget-wrapper';
+      const addContentBtnSelector = `${selector} [data-apos-add-content]`;
+      const videoBtnSelector = `${selector} [data-apos-add-item=apostrophe-video]`;
+      const videoInput = `.apos-field-input[name=video]`;
+      const videoUrl = 'https://www.youtube.com/watch?v=OV1yagf8Uqk';
+      const save = '[data-apos-save]';
+      client.getLocationInView(selector);
+      client.moveToElement(addContentBtnSelector, 0, 0);
+      client.waitForElementReady(addContentBtnSelector);
+      client.clickWhenReady(addContentBtnSelector);
+      client.waitForElementReady(videoBtnSelector);
+      client.pause(200);
+      client.clickWhenReady(videoBtnSelector);
+      client.waitForModal('apostrophe-video-widgets-editor');
+      client.resetValueInModal('apostrophe-video-widgets-editor', videoInput, videoUrl);
+      client.pause(3000);
+      client.clickInModal('apostrophe-video-widgets-editor', save);
+      client.waitForNoModals();
+    }
+  },
+  {
+    'Switch to human persona': function(client) {
+      const aposUi = '[data-apos-widget-wrapper=apostrophe-video] .apos-area-widget-controls'
+      const personaSelect = `${aposUi} .apos-button[name=personas]`;
+      const humanPersonaValue = `${personaSelect} option[value="human"]`;
+      client.execute(function(aposUi) {
```

Comment:
It looks like you had to do some jQuery gymnastics to work with these buttons. Can you factor that stuff out into a reusable step or command in apostrophe-nightwatch?

Reply:
Indeed. A step for displaying Apostrophe chrome (verse trying to get the mouse to hover over the element) sounds helpful and re-usable.